src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private int bound = 100;
```

Comment:
All magic numbers in your code should be **constants**.
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names
```suggestion
    private static final int MAX_BALL_NUMBER = 100;
    private Random random = new Random();
    private ColorSupplier colorSupplier = new ColorSupplier();    
```